testkit/src/main/scala/org/allenai/common/testkit/SingeInstanceUnitSpec.scala

```diff
@@ -0,0 +1,6 @@
+package org.allenai.common.testkit
+
+/** Base class for any unit test that, for some reason, want to share the Spec instance.
+  * Usually the reason is a slow expensive test setup time (setting up a 3rd party DB, etc.)
+  */
+abstract class SingeInstanceUnitSpec extends AllenAiBaseSpec
```

Comment:
@markschaake @jkinkead Does this start to sound more like an integration test? It's cool we can reuse Scala test infrastructure for both unit and it, I just wonder whether we want to make the distinction more clear via the name of the base classes as well.

Something like IntegrationSpec. <end brainstorm>